src/components/MessageInput.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useRef } from 'react';
+import { IMessage_Date, ITodo } from '../interfaces';
+
+type MessageInputProps = {
+  todos: ITodo[],
+  addMessage(msg: IMessage_Date): void
+}
+
+export const MessageInput: React.FC<MessageInputProps> = ({
+  todos,
+  addMessage,
+}) => {
+
+  const setNewMessage = (input_text:string): void => {
+    const rightNow = new Date();
+    const time = rightNow.toString().split('').splice(16, 5);
+    const timeString = time[0] + time[1] + time[2] + time[3] + time[4];
```

Comment:
you can use better approach: install inside of npm `moment.js` package and it can create date, convert date and format it easy